old-tutorials-in-development/tutorials-in-development/R/dc-spatio-temporal-intro/05-geographic-vs-projected-CRS.Rmd

```diff
@@ -212,52 +231,56 @@ the UTM zone, to avoid negative Easting numbers.
+    <figcaption>The 0,0 origin of each UTM zone is located in the **bottom left** hand corner (south west) of the zone - exactly 500,000 m EAST from the central meridian of the zone.  
     Source: Penn State E-education</figcaption>
 </figure>
 
 ***
 
 ### More on UTM
 
-* <a href="https://www.e-education.psu.edu/geog482spring2/c2_p22.html" target="_blank">
+* <a href="https://www.e-education.psu.edu/natureofgeoinfo/c2_p22.html" target="_blank">
 Penn State E-education overview of UTM</a>
-* <a href="https://www.e-education.psu.edu/geog482spring2/c2_p23.html
-" target="_blank">
+* <a href="https://www.e-education.psu.edu/natureofgeoinfo/c2_p23.html" target="_blank">
 More about UTM Zones - Penn State E-education</a>
 
 ***
 
 <figure>
-    <a href="https://upload.wikimedia.org/wikipedia/commons/thumb/e/ed/Utm-zones.jpg/800px-Utm-zones.jpg">
-    <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/ed/Utm-zones.jpg/800px-Utm-zones.jpg">
+	<a href="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/geospatial-skills/gridded-Utm-zones.jpg">
+	<img src="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/geospatial-skills/gridded-Utm-zones.jpg" alt="A world map with a grid overlay. The grid is labeled 1 to 60 going east and C to X going north.">
     </a>
     <figcaption> The gridded UTM coordinate system across the globe.    
     Source: Public domain from Earth Observatory, NASA</figcaption>
 </figure>
 
-# edit this messy
-The coordinates for the NEON Harvard Forest Field Site would be written as:
+### Understanding UTM Coordinates
+
+Let's look at the coordinates for the NEON Harvard Forest Field Site. The UTM coordinates 
+are written as:
 
 UTM Zone 18N, 730782m, 4712631m
 
-where the N denotes that it is in the Northern, not Southern hemisphere. Occassionally, you may see: 
+The N denotes that this point is in the Northern, not Southern, hemisphere. Occassionally, 
+you may see: 
 
-Zone 18T, 730782m Easting, 4712631m Northing.
+Zone 18T, 730782m Easting, 4712631m Northing
 
 <i class="fa fa-star"></i> **Data Tip:**  The UTM system doesn't apply to polar
 regions (>80°N or S). Universal Polar Stereographic (UPS) coordinate system is
-used in these area. This is where zones A, B and Y, Z are used if you were
+used in these areas. This is where zones A, B and Y, Z are used if you were
 wondering why they weren't in the UTM lettering system.  
-{: .notice }
 
 <div id="challenge" markdown="1">
 
-Practice understanding UTM coordinates:
+### Challenge: Practice understanding UTM coordinates
+
+The Penn State E-Education Institute has an interactive tool that you can use to explore UTM coordinate reference systems.
+
+<a href="https://www.e-education.psu.edu/natureofgeoinfo/sites/www.e-education.psu.edu.natureofgeoinfo/files/flash/coord_practice_utm_v06.swf" target="_blank">Download the UTM interactive tool</a>. Note: You will need Adobe Flash Player to use this tool.
```

Comment:
@donal-at-NEON: I got warning messages when I tried to download this: “this type of file can harm your computer”. I downloaded it anyway but couldn’t get flash working to use this file. So I’m not sure how useful it is, especially if tutorial users get that scary warning about the file.  Penn State's excellent resources are listed above (lines 242-246) so this challenge section may not be needed.

Reply:
Thanks for the explanation - sounds like we can delete this challenge section as I agree that 1) that's a scary warning, and 2) students can access those resources above